website/tables/aws/aws_ssm_document_versions.md

```diff
@@ -0,0 +1,33 @@
+# Table: aws_ssm_document_versions
+
+This table shows data for Ssm Document Versions.
```

Comment:
Could we add an exception for SSM here: https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/resources/plugin/plugin.go#L16 and then rerun doc generation 🙇